src/components/Navigation.js

```diff
@@ -4,8 +4,15 @@ import { NavLink } from 'react-router-dom';
 export default function Navigation() {
   return (
     <Navbar>
-      <Link to="/" aria-label="home">
-        <svg width="34" height="34" fill="#EDF8FF" viewBox="0 0 16 16">
+      <Link to="/" aria-label="Home">
+        <svg
+          role="img"
+          aria-label="House icon"
+          width="34"
+          height="34"
+          fill="#EDF8FF"
+          viewBox="0 0 16 16"
+        >
```

Comment:
As far as I know, the `aria-label` of the `Link` will be the name of your link and the `aria-label` of your svg would be ignored. And I think the `title` of an svg works as the name of an svg so you don't actuall need the `aria-label` here.
```suggestion
      <Link to="/" aria-label="Home">
        <svg
          role="img"
          width="34"
          height="34"
          fill="#EDF8FF"
          viewBox="0 0 16 16"
        >
```
Same for all the links. You could for example test this via your `.spec.js` with `screen.getByRole('link', { name: 'Home' })`.

Reply:
Yes i checked and change it, thx.